src/ilcdlib/epd/test_epd_reader.py

```diff
@@ -58,6 +60,31 @@ def test_read_basic_fields_industry_epd(self):
         self.assertEqual(self.epd_reader_industry.get_program_operator_id(), "EPD-HAM-20220202-ICD1-DE")
         self.assertEqual(self.epd_reader_industry.get_date_published(), datetime.date.fromisoformat("2022-10-10"))
 
+    def test_read_basic_fields_scenario_epd(self):
+        self.assertEqual(
+            self.epd_reader_industry.get_scenario_names(self.LANG), {'S1': '100% recycling', 'S2': 'Scenario 2'}
+        )
+
+    def test_read_epditaly_fields_scenario(self):
+        epditaly_reader = EpdItalyIlcdXmlEpdReader(
+            "2eb43850-0ab2-4068-afe5-218d69a096f8",
+            "00.01.000",
+            ZipIlcdReader(self.TEST_DATA_BASE / "ibu_with_dependencies.zip"),
+        )
+        self.assertEqual(
+            epditaly_reader.get_scenario_names(self.LANG), {'S1': 'S1', 'S2': 'S2'}
```

Comment:
The values like S1, S2 makes no sense actually. The value we want is something which makes clear what exact scenario the numbers are for. E.g. Recycling, Demolition, etc. Can you check if EPD etaly provides something like that?